diff --git a/epan/dissectors/CMakeLists.txt b/epan/dissectors/CMakeLists.txt
index dc11d09e18..abd1caa4c6 100644
--- a/epan/dissectors/CMakeLists.txt
+++ b/epan/dissectors/CMakeLists.txt
@@ -199,6 +199,7 @@ set(CLEAN_ASN1_DISSECTOR_SRC
 )
 
 set(DIRTY_ASN1_DISSECTOR_SRC
+	${CMAKE_CURRENT_SOURCE_DIR}/packet-j2735.c
 )
 
 set(ASN1_DISSECTOR_SRC
diff --git a/epan/dissectors/asn1/CMakeLists.txt b/epan/dissectors/asn1/CMakeLists.txt
index 5aec3bcb4c..cebbff3668 100644
--- a/epan/dissectors/asn1/CMakeLists.txt
+++ b/epan/dissectors/asn1/CMakeLists.txt
@@ -62,6 +62,7 @@ set(ASN1_SRC_DIRS
 	inap
 	isdn-sup
 	its
+	j2735
 	kerberos
 	lcsap
 	ldap
